src/main/scala/org/labrad/Server.scala

```diff
@@ -262,13 +264,58 @@ object Server {
    * Includes basic parsing of command line options.
    */
   def run(s: Server[_, _], args: Array[String]): Unit = {
-    val options = Util.parseArgs(args, Seq("name", "host", "port", "password"))
+    val parser = new ArgotParser("labrad",
```

Comment:
It might be easier to read if this were in a separate method (defineParameters?)